### PR TITLE
Add lodash alternatives

### PR DESCRIPTION
**Pull request by zeeshanu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1358,6 +1452,29 @@ Retrieves all the given object's own enumerable property values.
...
+Gets the value at path of object.
+
+  ```js
+  // Lodash
+  var object = { a: [{ b: { c: 3 } }] };
+  var result = _.get(object, 'a[0].b.c', 1);
+  console.log(result);
+  // output: 3
+
+  // Native
```
Comment (cht8687) [inline README.md]: Need to consider that Lodash's _.get can have default values if the value doesn't exist.
Object destructure assumes the path is valid for an object.
like below example, if `c` doesn't exist, it will generate error using this approach.

File: README.md [hunk 2]
```diff
@@ -128,13 +129,17 @@ then Lodash/Underscore is the better option.*
 **[Lang](#lang)**
 
 1. [_.isNaN](#_isnan)
+1. [_.isUndefined](#_isundefined)
+1. [_.gt](#_gt)
+1. [_.gte](#_gte)
```
Comment (terrierscript) [inline README.md]: Is those (`isUndefined / gt / gte / isArrayBuffer`) related `get` ?

I think better that split to another pull request if those not related.

File: README.md (hunk 1 above)
Comment (zeeshanu) [reply README.md]: @cht8687 Yes, you are right but this is the only way to replace the lodash's `get` method with vanilla JS. But we can also put a note about that specific issue.

Otherwise the only "ugly" way we can state is:

```javascript
let a = object.a || {};
a.b= a.b || {};

let objValue = a.b.c || 'default';
```

Comment (cht8687) [reply README.md]: @zeeshanu agreed. Put a note about this issue is fine.

File: README.md (hunk 2 above)
Comment (cht8687) [reply README.md]: @zeeshanu Could you separate them into different PR like what @terrierscript  suggested? thanks. :smile:

**Comment by cht8687** [comment]
@zeeshanu there are some conflicts, could you please resolve them before we can merge? cheers.

**Comment by zeeshanu** [comment]
@cht8687 Resolved. Could you please check again

**Comment by cht8687** [comment]
@zeeshanu  seems still have conflicts. Have you pull from latest master?

**Comment by zeeshanu** [comment]
@cht8687 Yes I did resolve all of them

**Comment by cht8687** [comment]
@zeeshanu You might need to update your upstream. Here it still showing "This branch cannot be rebased due to conflicts"
